recipes/_ghost.rb

```diff
@@ -4,6 +4,10 @@
     not_if { ::File.exist?("#{Chef::Config[:file_cache_path]}/ghost.zip") }
 end
 
+directory node['ghost-blog']['install_dir'] do
```

Comment:
If you could just update the check in L#14 then I would be good with merging this in.